From: Didier Raboud <odyx@debian.org>
Date: Tue, 9 Aug 2016 18:11:21 +0200
Subject: Tests: Use 127.0.0.1 instead of localhost to help pbuilder

This should make the test-suite work in pbuilder. It apparently also fixes an
ipp backend problem in the test-suite.

Origin: vendor
Patch-Name: tests-use-ipv4-lo-address.patch
---
 test/5.1-lpadmin.sh   | 4 ++--
 test/run-stp-tests.sh | 8 ++++----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/test/5.1-lpadmin.sh b/test/5.1-lpadmin.sh
index 7efc398..08b8c50 100644
--- a/test/5.1-lpadmin.sh
+++ b/test/5.1-lpadmin.sh
@@ -52,8 +52,8 @@ echo ""
 
 echo "Add Shared Printer Test"
 echo ""
-echo "    lpadmin -p Test3 -E -v ipp://localhost:$IPP_PORT/printers/Test2 -m everywhere"
-$runcups $VALGRIND ../systemv/lpadmin -p Test3 -E -v ipp://localhost:$IPP_PORT/printers/Test2 -m everywhere 2>&1
+echo "    lpadmin -p Test3 -E -v ipp://127.0.0.1:$IPP_PORT/printers/Test2 -m everywhere"
+$runcups $VALGRIND ../systemv/lpadmin -p Test3 -E -v ipp://127.0.0.1:$IPP_PORT/printers/Test2 -m everywhere 2>&1
 if test $? != 0; then
 	echo "    FAILED"
 	exit 1
diff --git a/test/run-stp-tests.sh b/test/run-stp-tests.sh
index 173fc86..88bafb1 100755
--- a/test/run-stp-tests.sh
+++ b/test/run-stp-tests.sh
@@ -490,7 +490,7 @@ fi
 cat >$BASE/cupsd.conf <<EOF
 StrictConformance Yes
 Browsing Off
-Listen localhost:$port
+Listen 127.0.0.1:$port
 Listen $BASE/sock
 MaxSubscriptions 3
 MaxLogSize 0
@@ -617,7 +617,7 @@ fi
 
 # These get exported because they don't have side-effects...
 CUPS_DISABLE_APPLE_DEFAULT=yes; export CUPS_DISABLE_APPLE_DEFAULT
-CUPS_SERVER=localhost:$port; export CUPS_SERVER
+CUPS_SERVER=127.0.0.1:$port; export CUPS_SERVER
 CUPS_SERVERROOT=$BASE; export CUPS_SERVERROOT
 CUPS_STATEDIR=$BASE; export CUPS_STATEDIR
 CUPS_DATADIR=$BASE/share; export CUPS_DATADIR
@@ -743,10 +743,10 @@ for file in 4*.test ../examples/ipp-2.1.test; do
         echo $ac_n "`date '+[%d/%b/%Y:%H:%M:%S %z]'` $ac_c" >>$strfile
 
 	if test $file = ../examples/ipp-2.1.test; then
-		uri="ipp://localhost:$port/printers/Test1"
+		uri="ipp://127.0.0.1:$port/printers/Test1"
 		options="-V 2.1 -d NOPRINT=1 -f testfile.ps"
 	else
-		uri="ipp://localhost:$port/printers"
+		uri="ipp://127.0.0.1:$port/printers"
 		options=""
 	fi
 	$runcups $VALGRIND ../tools/ipptool -tI $options $uri $file >> $strfile
